react-native.config.js

```diff
@@ -1,6 +1,37 @@
+let supportsCodegenConfig = true;
+
+// try {
+//   const rnCliAndroidVersion = require('@react-native-community/cli-platform-android/package.json')
+//     .version;
+//   const [major] = rnCliAndroidVersion.split('.');
+//   supportsCodegenConfig = major >= 9;
+// } catch (e) {
+//   // ignore
+// }
+
+// // this is a quick workaround for enabling custom codegen config in the fabric example app
+// // the require above picks up the version of the cli-platform-android package from the root node_modules
+// // instead of the one from the example app and doesn't correctly set the flag
+// // this, along with the patch to the rncli allows to run the fabric example properly
+// const codegenSwitch = '--enable-rnandroidprogressbar-codegen';
+// if (process.argv.includes(codegenSwitch)) {
```

Comment:
Is this needed?

Reply:
Yes, I uncommented it in https://github.com/react-native-progress-view/progress-bar-android/pull/90/commits/e1e2ab272cf2da97eeff7a823a4f171be4e470ae. It's there so we can have both the `example` and `fabricexample` apps working, since the CLI picked up in the `react-native.config.js` is the one from the root directory instead of `fabricexample` it doesn't use the custom codegen config. This way we can have it working in both with a simple patch to the CLI adding a flag.